Plugin~/WebRTCPlugin/UnityVideoRenderer.cpp

```diff
@@ -0,0 +1,57 @@
+#include "pch.h"
+#include "UnityVideoRenderer.h"
+
+namespace unity {
+namespace webrtc {
+
+UnityVideoRenderer::UnityVideoRenderer()
+{
+    //Do nothing
+    DebugLog("Create UnityVideoRenderer");
+}
+
+UnityVideoRenderer::~UnityVideoRenderer() = default;
+
+void UnityVideoRenderer::OnFrame(const webrtc::VideoFrame &frame)
+{
+    rtc::scoped_refptr<webrtc::VideoFrameBuffer> frame_buffer = frame.video_frame_buffer();
```

Comment:
`video_frame_buffer` is not deep copy.
The `frame_buffer` should be disposed when removed from the method scope.

I think `frame_buffer::ToI420()` method is copying buffer so it might be no problem.
But if `frame_buffer::type()` do not return `kNative`, it has possibility to happen uncertain behaviors.

Reply:
If i change to below code, this test is faild. https://github.com/Unity-Technologies/com.unity.webrtc/pull/181/files#diff-f623d07937d2a15dc825dee74c53a9deR70

```
void UnityVideoRenderer::OnFrame(const webrtc::VideoFrame &frame)
{
    rtc::scoped_refptr<webrtc::VideoFrameBuffer> frame_buffer = frame.video_frame_buffer();
    frame_buffer = frame_buffer->ToI420();
    SetFrameBuffer(frame_buffer);
}
```

I think `frame_buffer` is set local field on `SetFrameBuffer`.
So, it means add new reference